examples/gno.land/r/demo/privaterealm/privaterealm.gno

```diff
@@ -0,0 +1,7 @@
+package privaterealm
+
+func Render(path string) string {
+	// This realm is an example of a private realm.
+	// It cannnot be imported or have state persistence.
```

Comment:
Private realms can have state persistence. However, they cannot store anything that is locally defined or locally typed in another realm.